Jenkinsfile

```diff
@@ -1,9 +1,21 @@
 elifePipeline {
     def commit
-    stage 'Checkout', {
-        checkout scm
-        commit = elifeGitRevision()
-    }
+    elifeOnNode(
+        {
+            stage 'Checkout', {
+                checkout scm
+                commit = elifeGitRevision()
+            }
+
+            stage 'Container image', {
+                sh 'docker-compose build'
+                sh 'chmod 777 build/ && docker-compose -f docker-compose.ci.yml run ci ./project_tests.sh'
```

Comment:
777 is necessary because the container is then able to write build artifacts like XML test results in `build/`